src/components/views/right_panel/GroupHeaderButtons.js

```diff
@@ -68,13 +68,13 @@ export default class GroupHeaderButtons extends HeaderButtons {
             <HeaderButton key="groupMembersButton" name="groupMembersButton"
                 title={_t('Members')}
                 isHighlighted={this.isPhase(groupPhases)}
-                clickPhase={RightPanel.Phase.GroupMemberList}
+                onClick={() => this.togglePhase(RightPanel.Phase.GroupMemberList)}
```

Comment:
Shouldn't these be handlers in the component to avoid creating new functions every render?

Reply:
yeah, was on the verge to create dedicated handler methods. Will do so :+1: